test/postgresql.test.js

```diff
@@ -263,6 +263,18 @@ describe('postgresql connector', function() {
       });
   });
 
+  it('should support where filter for array type field', function(done) {
+    Post.dataSource.settings.allowExtendedOperators = true;
+    Post.find({where: {and: [
+      {
+        categories: {'@>': ['AA']},
+      },
+    ]}}, function(err, post) {
+      should.not.exist(err);
```

Comment:
```suggestion
      if (err) return done(err);
```

That way the original error details and the stack trace is preserved.

Reply:
How we would be handling this with async await type test case? @bajtos 
I was thinking to write test case this way for handling failure of testcase with proper details.


it('should support where filter for array type field', async () => {
      Post.dataSource.settings.allowExtendedOperators = true;
      try {
        const post = await Post.find({where: {
          categories: {'contains': ['AA']},
        }});
        should.exist(post);
        post.length.should.equal(1);
      } catch (err) {
        should.not.exist(err);
      }
    });


Please tell me if it is correct, I will update current test case in this way only.

Reply:
As a rule of thumb, the tests should pass any unexpected error to the test runner:
- Use `done(err)` when using callback
- Use regular error propagation when using async functions

```js
it('should support where filter for array type field', async () => {
      Post.dataSource.settings.allowExtendedOperators = true;
        const post = await Post.find({where: {
          categories: {'contains': ['AA']},
        }});
        should.exist(post);
        post.length.should.equal(1);
    });
```